src/scripts/towns/TownList.ts

```diff
@@ -5240,6 +5265,44 @@ const Calem2 = new NPC('Calem', [
     requirement: new MultiRequirement([new QuestLineStepCompletedRequirement('A Beautiful World', 1), new QuestLineStepCompletedRequirement('A Beautiful World', 3, GameConstants.AchievementOption.less)]),
 });
 
+const KalosFossilNpc1 = new NPC('Honeymooner Gal', [
+    'I must find one! I simply must!',
+    'I sent my husband to buy tickets for the aquarium but it was all a ploy to get some alone time so I can find him a Pokémon fossil found only here in Kalos!',
+    'According to my brochure, there\'s a pretty cave past these cliffs that\'s filled with them! It says reviving a Jaw Fossil can give you a Tyrunt, the perfect anniversary gift for my sweetie pie!',
+    'I WILL find a Jaw Fossil! Nothing will get in the way of seeing my booboo\'s smile! Anything is possible with the power of love! And don\'t you forget that either!',
+], {
+    image: 'assets/images/npcs/Tourist (female).png',
+    requirement: new OneFromManyRequirement([
+        new ObtainedPokemonRequirement('Tyrunt', true),
+        new ObtainedPokemonRequirement('Amaura', true),
+    ]),
+});
+
+const KalosFossilNpc2 = new NPC('Honeymooner Guy', [
+    'I have to find one! I just have to!',
+    'I told my wife I was going buy tickets for the aquarium but little did she know it was a white lie so I could search for a Pokémon fossil found only here in Kalos!',
+    'According to my travel guide, this cave right here is filled with them! It says reviving a Sail Fossil can nab you an Amaura, the perfect anniversary gift for my pumpkin!',
+    'I MUST find a Sail Fossil! Nothing will get in the way of making my sunshine happy! She\'s taught me that anything is possible with the power of love!',
+], {
+    image: 'assets/images/npcs/Tourist (male).png',
+    requirement: new OneFromManyRequirement([
+        new ObtainedPokemonRequirement('Tyrunt', true),
+        new ObtainedPokemonRequirement('Amaura', true),
```

Comment:
I feel like one of them should go away, when you find one of the pokemon? Like, if one of them have found their fossil, they should no longer be looking for it

Reply:
That is intended, since they never find their fossils. It's entirely dependent on the player's progress. I've tweaked the dialogue a bit to reflect this intention.